src/components/App.tsx

```diff
@@ -317,6 +319,18 @@ export class App extends React.Component<AppProps, AppState> {
     const projectModel = this.state.project.getModel();
     this.controlCenter.sandbox.run(projectModel, src);
   }
+
+  showPreview(file: File) {
+    if (file.type === FileType.Markdown) {
+      addFilePreview(file);
+      const { preview } = file;
+
+      // TODO: Move splitting logic outside of App.tsx
```

Comment:
we don't need this TODO comment, I'm not sure what it means.

nit: can you also remove empty line above to make `const { preview } ` close to the next line.